src/views/Home/components/Schedule/index.view.tsx

```diff
@@ -40,12 +64,12 @@ const ScheduleComponent: React.FC = () => {
           </div>
           <div className="Schedule__greyRect"></div>
           <div className="Schedule__eventContainer">
```

Comment:
good DOM structure here! to remove the last rule line from rendering in this list of schedule events, [use the `:last-child` selector](https://css-tricks.com/almanac/selectors/l/last-child/)